### ISSUE TITLE
Cannot save the site `https://n0.computer`

### ISSUE THREAD

**Issue by yhm-amber** [gildas-lormeau/SingleFile#1081]
### Describe the bug

Cannot save the site `https://n0.computer`, and just get `err` message so cannot find the reason ...

**Only this site will err.**

### To Reproduce

1. Go to site [n0.computer](https://n0.computer)
2. Click on `SingleFile button`
3. wait

### Expected behavior

save success or show me some reason

### Screenshots

![image](https://user-images.githubusercontent.com/103625580/209523679-5ab2cbe5-6722-493a-ab35-7b124c61408d.png)

### Environment

 - OS: `Win10 Family` ver `22H2`
 - Browser: `brave` ver `1.48.83  Chromium: 109.0.5414.46`
 - SingleFile Version: `1.21.29` id `mpiodijhokgodhhofbcjdecpffjipkle`

### Additional context

<details>

<summary>my profile</summary>

~~~ json
{
  "profiles": {
    "__Default_Settings__": {
      "acceptHeaders": {
        "audio": "audio/webm,audio/ogg,audio/wav,audio/*;q=0.9,application/ogg;q=0.7,video/*;q=0.6,*/*;q=0.5",
        "document": "text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8",
        "font": "application/font-woff2;q=1.0,application/font-woff;q=0.9,*/*;q=0.8",
        "image": "image/avif,image/webp,image/apng,image/svg+xml,image/*,*/*;q=0.8",
        "script": "*/*",
        "stylesheet": "text/css,*/*;q=0.1",
        "video": "video/webm,video/ogg,video/*;q=0.9,application/ogg;q=0.7,audio/*;q=0.6,*/*;q=0.5"
      },
      "addProof": false,
      "allowedBookmarkFolders": [
        ""
      ],
      "applySystemTheme": true,
      "autoClose": false,
      "autoOpenEditor": false,
      "autoSaveDelay": 1,
      "autoSaveDiscard": false,
      "autoSaveExternalSave": false,
      "autoSaveLoad": false,
      "autoSaveLoadOrUnload": true,
      "autoSaveRemove": false,
      "autoSaveRepeat": false,
      "autoSaveRepeatDelay": 10,
      "autoSaveUnload": false,
      "backgroundSave": true,
      "blockAudios": true,
      "blockFont": false,
      "blockFonts": false,
      "blockImages": false,
      "blockMixedContent": false,
      "blockScripts": true,
      "blockStylesheets": false,
      "blockVideos": true,
      "browserActionMenuEnabled": true,
      "compressCSS": true,
      "compressHTML": true,
      "confirmFilename": true,
      "confirmInfobarContent": false,
      "contextMenuEnabled": true,
      "defaultEditorMode": "normal",
      "displayInfobar": true,
      "displayStats": true,
      "filenameConflictAction": "uniquify",
      "filenameMaxLength": "1024",
      "filenameMaxLengthUnit": "bytes",
      "filenameReplacedCharacters": [
        "~",
        "+",
        "\\\\",
        "?",
        "%",
        "*",
        ":",
        "|",
        "\"",
        "<",
        ">",
        "\u0000-\u001f",
        ""
      ],
      "filenameReplacementCharacter": "_",
      "filenameTemplate": "sigfils_archives/{year-locale}/{month-locale}/{url-last-segment},{time-ms}-{visit-time-ms},{page-title}[192],d.html",
      "forceWebAuthFlow": false,
      "githubBranch": "main",
      "githubRepository": "SingleFile-Archives",
      "githubToken": "",
      "githubUser": "",
      "groupDuplicateImages": true,
      "ignoredBookmarkFolders": [
        ""
      ],
      "includeBOM": false,
      "includeInfobar": true,
      "infobarTemplate": "",
      "insertMetaCSP": true,
      "insertMetaNoIndex": false,
      "insertSingleFileComment": true,
      "loadDeferredImages": true,
      "loadDeferredImagesBlockCookies": false,
      "loadDeferredImagesBlockStorage": false,
      "loadDeferredImagesDispatchScrollEvent": true,
      "loadDeferredImagesKeepZoomLevel": false,
      "loadDeferredImagesMaxIdleTime": 1500,
      "logsEnabled": true,
      "maxResourceSize": 10,
      "maxResourceSizeEnabled": false,
      "maxSizeDuplicateImages": 524288,
      "moveStylesInHead": false,
      "networkTimeout": 0,
      "openEditor": false,
      "openSavedPage": false,
      "passReferrerOnError": false,
      "progressBarEnabled": true,
      "removeAlternativeFonts": true,
      "removeAlternativeImages": true,
      "removeAlternativeMedias": false,
      "removeFrames": true,
      "removeHiddenElements": true,
      "removeImports": true,
      "removeUnusedFonts": false,
      "removeUnusedStyles": true,
      "replaceBookmarkURL": true,
      "resolveFragmentIdentifierURLs": false,
      "saveCreatedBookmarks": false,
      "saveFavicon": true,
      "saveOriginalURLs": true,
      "saveRawPage": false,
      "saveToClipboard": false,
      "saveToGDrive": false,
      "saveToGitHub": false,
      "saveWithCompanion": false,
      "saveWithWebDAV": false,
      "shadowEnabled": true,
      "tabMenuEnabled": true,
      "userScriptEnabled": false,
      "warnUnsavedPage": true,
      "webDAVPassword": "",
      "webDAVURL": "",
      "webDAVUser": "",
      "woleetKey": ""
    },
    "j": {
      "acceptHeaders": {
        "audio": "audio/webm,audio/ogg,audio/wav,audio/*;q=0.9,application/ogg;q=0.7,video/*;q=0.6,*/*;q=0.5",
        "document": "text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8",
        "font": "application/font-woff2;q=1.0,application/font-woff;q=0.9,*/*;q=0.8",
        "image": "image/avif,image/webp,image/apng,image/svg+xml,image/*,*/*;q=0.8",
        "script": "*/*",
        "stylesheet": "text/css,*/*;q=0.1",
        "video": "video/webm,video/ogg,video/*;q=0.9,application/ogg;q=0.7,audio/*;q=0.6,*/*;q=0.5"
      },
      "addProof": false,
      "allowedBookmarkFolders": [
        ""
      ],
      "applySystemTheme": true,
      "autoClose": false,
      "autoOpenEditor": false,
      "autoSaveDelay": 1,
      "autoSaveDiscard": false,
      "autoSaveExternalSave": false,
      "autoSaveLoad": false,
      "autoSaveLoadOrUnload": true,
      "autoSaveRemove": false,
      "autoSaveRepeat": false,
      "autoSaveRepeatDelay": 10,
      "autoSaveUnload": false,
      "backgroundSave": true,
      "blockAudios": false,
      "blockFont": false,
      "blockFonts": false,
      "blockImages": false,
      "blockMixedContent": false,
      "blockScripts": false,
      "blockStylesheets": false,
      "blockVideos": false,
      "browserActionMenuEnabled": true,
      "compressCSS": true,
      "compressHTML": true,
      "confirmFilename": false,
      "confirmInfobarContent": false,
      "contextMenuEnabled": true,
      "defaultEditorMode": "normal",
      "displayInfobar": true,
      "displayStats": false,
      "filenameConflictAction": "uniquify",
      "filenameMaxLength": "1024",
      "filenameMaxLengthUnit": "bytes",
      "filenameReplacedCharacters": [
        "~",
        "+",
        "\\\\",
        "?",
        "%",
        "*",
        ":",
        "|",
        "\"",
        "<",
        ">",
        "\u0000-\u001f",
        ""
      ],
      "filenameReplacementCharacter": "_",
      "filenameTemplate": "sigfils_archives/{year-locale}/{month-locale}/{url-last-segment},{time-ms}-{visit-time-ms},{page-title}[192],j.html",
      "forceWebAuthFlow": false,
      "githubBranch": "main",
      "githubRepository": "SingleFile-Archives",
      "githubToken": "",
      "githubUser": "",
      "groupDuplicateImages": true,
      "ignoredBookmarkFolders": [
        ""
      ],
      "includeBOM": false,
      "includeInfobar": true,
      "infobarTemplate": "",
      "insertMetaCSP": true,
      "insertMetaNoIndex": false,
      "insertSingleFileComment": true,
      "loadDeferredImages": true,
      "loadDeferredImagesBlockCookies": false,
      "loadDeferredImagesBlockStorage": false,
      "loadDeferredImagesDispatchScrollEvent": true,
      "loadDeferredImagesKeepZoomLevel": false,
      "loadDeferredImagesMaxIdleTime": 1500,
      "logsEnabled": true,
      "maxResourceSize": 12,
      "maxResourceSizeEnabled": true,
      "maxSizeDuplicateImages": 524288,
      "moveStylesInHead": false,
      "networkTimeout": 0,
      "openEditor": false,
      "openSavedPage": false,
      "passReferrerOnError": false,
      "progressBarEnabled": true,
      "removeAlternativeFonts": true,
      "removeAlternativeImages": true,
      "removeAlternativeMedias": false,
      "removeFrames": true,
      "removeHiddenElements": false,
      "removeImports": true,
      "removeUnusedFonts": false,
      "removeUnusedStyles": true,
      "replaceBookmarkURL": true,
      "resolveFragmentIdentifierURLs": false,
      "saveCreatedBookmarks": false,
      "saveFavicon": true,
      "saveOriginalURLs": true,
      "saveRawPage": false,
      "saveToClipboard": false,
      "saveToGDrive": false,
      "saveToGitHub": false,
      "saveWithCompanion": false,
      "saveWithWebDAV": false,
      "shadowEnabled": true,
      "tabMenuEnabled": true,
      "userScriptEnabled": false,
      "warnUnsavedPage": true,
      "webDAVPassword": "",
      "webDAVURL": "",
      "webDAVUser": "",
      "woleetKey": ""
    }
  },
  "rules": [
    {
      "autoSaveProfile": "__Disabled_Settings__",
      "profile": "__Default_Settings__",
      "url": "file:"
    }
  ],
  "maxParallelWorkers": 11
}
~~~

</details>

**Comment by gildas-lormeau** [comment]
Thank you for the detailed report, it's fixed in the version 1.21.32 I just published.

**Comment by yhm-amber** [comment]
> Thank you for the detailed report, it's fixed in the version 1.21.32 I just published.

I see it! It fixed! Thx 😆

![image](https://user-images.githubusercontent.com/103625580/214556385-78e6137e-259a-4cfd-b76f-d0ff3167e70a.png)
